desktop/core/src/desktop/api2.py

```diff
@@ -1418,6 +1418,21 @@ def _paginate(request, queryset):
   return __paginate(page, limit, queryset)
 
 
+@api_error_handler
+def available_app_examples(request):
+  """Returns a dictionary of available apps whose examples can be installed by the admin user."""
+
+  if not is_admin(request.user):
+    return JsonResponse({'message': "You must be a Hue admin to access this endpoint."}, status=403)
+
+  supported_app_examples = {'hive', 'impala', 'hbase', 'pig', 'oozie', 'notebook', 'search', 'spark', 'jobsub'}
+
+  # Filter apps based on supported list
+  available_examples = {app.name: app.nice_name for app in appmanager.get_apps(request.user) if app.name in supported_app_examples}
```

Comment:
are we checking for blacklist here? does get_apps take care of it?

Reply:
get_apps take care of it